.github/workflows/backend-flaky-tests-detection.yaml

```diff
@@ -0,0 +1,194 @@
+
+name: Backend Flaky Tests Analysis
+
+on:
+  schedule:
+    - cron: '0 0 * * 1' # Runs every monday at 00:00 AM UTC
+  workflow_dispatch:
+    inputs:
+      num_containers:
+        description: "Number of containers to use (Max: 200) (Default: 10)"
+        required: false
+
+env:
+  MIX_ENV: test
+  NODE_VERSION: "20"
+  PYTHON_VERSION: "3.12.3"
+  DEFAULT_NUM_CONTAINERS: 10
+
+jobs:
+  elixir-deps:
+    name: Elixir ${{ matrix.mix_env }} dependencies
+    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        include:
+          - mix_env: dev
+          - mix_env: test
```

Comment:
It's probably because of monday morning, but I am not understanding why we're preparing deps for both dev and test environments 👀

Reply:
if we are going to only cover backend tests, `test` env should be enough.

And if we're not going to run frontend related things, yes npm deps might be not necessary. But better try it 😄